### PR TITLE
category select arrow fix

### PR DESCRIPTION
**Pull request by tanjimhasan**

### CODE REVIEW

**Comment by tanjimhasan** [comment]
https://prnt.sc/fbbFU_WH8qYi
issue fixed

File: assets/src/js/global/components/select2-custom-control.js [hunk 1]
```diff
@@ -25,7 +25,7 @@ function setup_dom_observer() {
         // Create an observer instance linked to the callback function
         const observer = new MutationObserver( () => {
 
-            setTimeout( init, 200 );
+            setTimeout( init, 300 );
```
Comment (kowsar89) [inline assets/src/js/global/components/select2-custom-control.js:28]: why settimeout is used? is it a good practice? @tanjimhasan

**Review by M4hd1BD — COMMENTED** [review]
Need Improvements

**Review by kowsar89 — CHANGES_REQUESTED** [review]
why package-lock.json is updated? make sure node version is same when building assets

**Comment by kowsar89** [comment]
Works fine,  confirmed by Tayar

**Comment by kowsar89** [comment]
closed on the favour of https://github.com/sovware/directorist/pull/1000

**Pull request closed without merging**